controllers/rabbitmqcluster_controller_test.go

```diff
@@ -863,10 +867,10 @@ var _ = Describe("RabbitmqClusterController", func() {
 			waitForClusterCreation(ctx, cluster, client)
 		})
 
-		AfterEach(func() {
-			Expect(client.Delete(ctx, cluster)).To(Succeed())
-			waitForClusterDeletion(ctx, cluster, client)
-		})
+		// AfterEach(func() {
+		// 	Expect(client.Delete(ctx, cluster)).To(Succeed())
+		// 	waitForClusterDeletion(ctx, cluster, client)
+		// })
```

Comment:
Did you meant to leave this bit commented out?

Reply:
Good catch, I removed it